core/src/main/scala/kafka/tools/ConsoleProducer.scala

```diff
@@ -135,126 +136,135 @@ object ConsoleProducer {
 
   class ProducerConfig(args: Array[String]) {
     val parser = new OptionParser(false)
-    val topicOpt = parser.accepts("topic", "REQUIRED: The topic id to produce messages to.")
+    val topicOpt = parser.accepts("topic", "The topic id to produce messages to.")
       .withRequiredArg
-      .describedAs("topic")
+      .describedAs("topic id")
       .ofType(classOf[String])
-    val brokerListOpt = parser.accepts("broker-list", "REQUIRED: The broker list string in the form HOST1:PORT1,HOST2:PORT2.")
+      .required
+    val brokerListOpt = parser.accepts("broker-list", "The broker list string in the form host1:port1,host2:port2.")
       .withRequiredArg
-      .describedAs("broker-list")
+      .describedAs("server(s) to connect to")
       .ofType(classOf[String])
-    val syncOpt = parser.accepts("sync", "If set message send requests to the brokers are synchronously, one at a time as they arrive.")
-    val compressionCodecOpt = parser.accepts("compression-codec", "The compression codec: either 'none', 'gzip', 'snappy', or 'lz4'." +
-                                                                  "If specified without value, then it defaults to 'gzip'")
+      .required
+    val syncOpt = parser.accepts("sync", "If set, message requests are send to the brokers synchronously, one at a time as they arrive.")
+    val compressionCodecOpt = parser.accepts("compression-codec", "The compression codec is either: 'none', 'gzip', 'snappy', or 'lz4'." +
+                                                                  "If specified without value, then it defaults to 'gzip'.")
                                     .withOptionalArg()
                                     .describedAs("compression-codec")
                                     .ofType(classOf[String])
     val batchSizeOpt = parser.accepts("batch-size", "Number of messages to send in a single batch if they are not being sent synchronously.")
       .withRequiredArg
-      .describedAs("size")
+      .describedAs("Number of messages in a single batch")
```

Comment:
nit: Looks like in most places these descriptions start with a lowercase letter.